data/human/free worlds epilogue.txt

```diff
@@ -197,3 +197,96 @@ mission "FW Epilogue: Danforth"
 			label end
 			`	After you talk for a while longer, Danforth wishes you luck and sends you on your way. "I'll be in touch if we ever need your assistance in the future," he says.`
 				decline
+
+
+
+mission "FW Epilogue: Barmy"
+	minor
+	landing
+	name `Weapon Testing`
+	description `A test ship with new weaponry is in orbit around <planet>. Fight and disable it (but do not board or destroy it!) and then land to give Barmy Edward feedback on its performance.`
+	source "Rust"
+	to offer
+		has "free worlds plot completed"
+	on offer
+		conversation
+			`As you complete your landing procedure, you recall having come here to assist in some testing for new weapons, back in the thick of the war. Do you want to check in on Barmy Edwards, the lead engineer?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					defer
+			scene "scene/loc0"
+			`	You give your name to the receptionist, and she directs you to a hangar where you find Barmy Edward, who is working on a weapon that looks like a modified plasma cannon. When he notices you, he gets up from his work and shakes your hand warmly. "Ah, <last>. It's been too long! How fortunate for me! I'm actually just putting the finishing touches on a new prototype. You wouldn't be too busy to assist me in some practical testing, would you?`
+			choice
+				`	"I'm not sure about this. What would you have me do?"`
+				`	"Being a target for experimental weapons? Just tell me what you need!"`
+			`	"Not much out of the ordinary from our tests in the past," he replies. "You'll find a ship of ours in orbit that goes by 'Hunk'. It's been fitted with a few of these new weapons. Your job will be to fight and disable it, tell us what you think of its combat capabilities, and transfer your ship's logs so we can analyze the fight."`
+			choice
+				`	"Alright, I suppose for old times' sake I can help out."`
+				`	"Sorry, I just wanted to check in and see how you were doing."`
+					goto catchup
+			`	He turns to you and smirks. "We've recommissioned the ship from retirement, so it's not top of the line, but try to keep it in one piece for future use. Understand?"`
+			choice
+				`	"Of course! I'll take care of it."`
+			`	"Great," he says, rubbing his hands together with a gleeful expression on his face. He's clearly looking forward to hearing what you think of his latest invention.`
+				accept
+			label catchup
+			`	"I see. Well, I'm sure we can find someone else up to the task. Regardless, Captain <last>, I assure you this new weapon vastly improves on the outdated plasma cannon. This'll put Kraz Cybernetics in the big leagues!"`
+			`	As his ramblings about his new invention continue, you think to yourself about the old times, the war, and all the people you've met along the way.`
```

Comment:
This line doesn't really seem like it fits here. I'd just toss it, maybe reuse it somewhere else if you can.
```suggestion
```